feature/orchestrator/src/main/AndroidManifest.xml

```diff
@@ -46,9 +45,9 @@
             <intent-filter>
                 <action android:name="com.simprints.id.REGISTER" />
                 <action android:name="com.simprints.id.IDENTIFY" />
-                <action android:name="com.simprints.id.UPDATE" />
                 <action android:name="com.simprints.id.VERIFY" />
                 <action android:name="com.simprints.id.CONFIRM_IDENTITY" />
+                <action android:name="com.simprints.commcare.REGISTER_LAST_BIOMETRICS" />
```

Comment:
Shouldn't it be `com.simprints.id` here?

Reply:
Yep, copy-paste error